docs/reference/islands.md

```diff
@@ -49,30 +53,18 @@ export default function InteractiveComponent({ defaultCount }) {
 
 ## Island props
 
-### `hydrateOn`
-
-When rendering a page with Islands on the server the output includes a script to initialize Islands on the client. The default behavior of the island initialization script is to eagerly hydrate all Islands as soon as possible, i.e., on load, but there are other strategies available when hydrating components.
-
-The available hydration types are:
-
-- `load` (default)
-- `idle`
-- `visible`
-
-These types can be set as the hydrateOn prop on the Island Component, for example:
-
-```javascript
-<Island module={InteractiveComponent} hydrateOn=”visible” />
-```
-
-For islands with the `idle` hydration type we use [requestIdleCallback](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback), allowing the hydration to be deferred. This is good for lower priority components, allowing client resources to be used first on higher priority.
-
-For Islands with the `visible` hydration type we don’t hydrate until the element is visible on screen by using the [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API). This mode is good for components that aren't visible to the user immediately, e.g., if they are further down a long page.
-
-Hydrating means downloading and initializing your Island component code, so using these different hydration types strategically to defer some of that work can help boost initial page load performance!
-
-### `clientOnly`
-
-`boolean`
-
-This is set to `false` by default, but when set to `true` then the Island won’t be rendered on the server. This can be useful for components that rely on logic/libraries that can only run in the browser.
+Name | Type | Default | Description
+-- | -- | -- | --
+<span style="color:white">**clientOnly**</span> | `boolean` | `false` | This is set to `false` by default, but when set to `true` then the Island won’t be rendered on the server. This can be useful for components that rely on logic/libraries that can only run in the browser.
+<span style="color:white">**hydrateOn**</span> | `"load" \| "visible" \| "idle"` | `"load"` | When rendering a page with Islands on the server the output includes a script to initialize Islands on the client. Hydrating means downloading and initializing the Island component code, so using these different hydration types strategically to defer some of that work can help boost initial page load performance! See [Hydration Types](#hydration-types) below for more information.
+<span style="color:white">**id**</span> | `string` | - | This value is passed through to the island's `wrapperTag` id HTML attribute.
```

Comment:
We generate a unique ID for an island `island-123456` if one is not provided to us by the developer.

Reply:
Ah, thanks for the context. What would be the purpose/benefit of a developer providing us their own `id` for the island instead of just using our generated one?

Reply:
@bmatto following up here before merging in this PR 😄. I don't understand the need to have devs pass their own id here.

Reply:
Some is historical (there was a point a while ago when our internals were not smart enough to have stable ID across server and client renders... so a unique ID was required if multiple island were in a module). But I think still there is "what if a dev wants to have their own ID they target for whatever style/etc reason" too 🤷

Reply:
Gotcha. That makes sense, I will keep it in the docs!